stock_picking_default_planned_transfer/hooks.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID
+from odoo.api import Environment
+
+
+def no_default_immediate_tranfer_create(cr, registry):
+    env = Environment(cr, SUPERUSER_ID, {})
+    config_parameter = env["ir.config_parameter"].search(
+        [("key", "=", "stock.no_default_immediate_tranfer")]
+    )
+    if config_parameter and config_parameter.value != "True":
+        config_parameter.write({"value": "True"})
+    else:
+        env["ir.config_parameter"].create(
+            {
+                "key": "stock.no_default_immediate_tranfer",
+                "value": "True",
+            }
+        )
```

Comment:
I think you could make the code easier using [set_param](https://github.com/OCA/OCB/blob/16.0/odoo/addons/base/models/ir_config_parameter.py#L80)

```suggestion
    env["ir.config_parameter"].set_param("stock.no_default_immediate_tranfer", "True")
```

Reply:
@petrus-v , I m not sure because stock.no_default_immediate_tranfer is called in Odoo native but not existing, we must create it IMO

Reply:
@Kev-Roche That's exactly what Odoo does: https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/models/ir_config_parameter.py#L95

Reply:
ho, did not know that! So, yes @petrus-v change is better. Thank you @rousseldenis